avro-builder/tests/codegen-14/build.gradle

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2022 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+plugins {
+  id "java-library"
+}
+
+configurations {
+  codegen
+}
+
+sourceSets {
+  main {
+    java {
+      srcDir 'src/main/java'
+      srcDir "$buildDir/generated/sources/avro/java/main"
+    }
+    resources {
+      srcDirs = [
+        "src/main/avro"
+      ]
+    }
+  }
+}
+
+task runOwnCodegen {
+  description = 'generate specific classes using own codegen utility'
+  doLast {
+    javaexec {
+      classpath configurations.codegen
+      main = 'com.linkedin.avroutil1.builder.SchemaBuilder'
+      args = [
+          "--input", "$projectDir/src/main/avro",
+          "--output", "$buildDir/generated/sources/avro/java/main"
+      ]
+    }
+  }
+}
+
+compileJava.dependsOn runOwnCodegen
+
+dependencies {
+  codegen project(":avro-builder:builder")
+
+  implementation ("org.apache.avro:avro:1.4.1") {
+    exclude group: "org.mortbay.jetty"
+    exclude group: "org.apache.velocity"
+    exclude group: "commons-lang"
+    exclude group: "org.jboss.netty"
+    exclude group: "com.thoughtworks.paranamer", module: "paranamer-ant"
+  }
+  //required because generated code depends on the helper
+  implementation project(":helper:helper")
+
+//  codegen "net.sf.jopt-simple:jopt-simple:3.2"   //avro 1.4 needs this for codegen but is missing a dependency
+//  codegen "org.apache.hadoop:hadoop-core:0.20.2" //avro 1.4 needs this for codegen but is missing a dependency
+//  codegen project(":helper:tests:helper-tests-common")
+//  codegen project(":helper:helper")
+//  //redirect logging to log4j2 for code generation
+//  codegen "org.apache.logging.log4j:log4j-core:2.17.1"
+//  codegen "org.apache.logging.log4j:log4j-slf4j-impl:2.17.1"
+//  codegen files('../codegenClasspath')
+//
+//  //required because generated code depends on the helper
+//  implementation project(":helper:helper")
+//  implementation ("org.apache.avro:avro:1.4.1") {
+//    exclude group: "org.mortbay.jetty"
+//    exclude group: "org.apache.velocity"
+//    exclude group: "commons-lang"
+//    exclude group: "org.jboss.netty"
+//    exclude group: "com.thoughtworks.paranamer", module: "paranamer-ant"
+//  }
+//
+//  //this block required for resource generation code
+//  implementation project(":helper:tests:helper-tests-common")
+//  //redirect logging to log4j2 for resource generation
+//  resourcegen "org.apache.logging.log4j:log4j-core:2.17.1"
+//  resourcegen "org.apache.logging.log4j:log4j-slf4j-impl:2.17.1"
+//  resourcegen files('../codegenClasspath')
```

Comment:
Should we remove commented-out code from this PR?

Reply:
yes. thanks for catching this.